sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -209,7 +214,7 @@ export class AssessmentController {
    * Worker to be executed at very end of the workflow responsible for latest tasks before the `end event`.
    */
   @ZeebeWorker(Workers.WorkflowWrapUp, {
-    fetchVariable: [ASSESSMENT_ID, WORKFLOW_DATA],
+    fetchVariable: [ASSESSMENT_ID, STUDENT_ID, PROGRAM_YEAR_ID, WORKFLOW_DATA],
```

Comment:
Not a blocker but the workers, as much as possible so far, receive the minimum information from the Workflow.
Almost all of them are driven by `assessmentId` or `applicationId` and from that information, they will retrieve data like `studentId` and `programYearId`.